collectors/rest/salesforce/collector.json

```diff
@@ -0,0 +1,68 @@
+{
+  "type": "collection",
+  "ttl": "4h",
+  "removeFields": [],
+  "resumeOnBoot": false,
+  "schedule": {},
+  "streamtags": [],
+  "workerAffinity": false,
+  "collector": {
+    "conf": {
+      "discovery": {
+        "discoverType": "none"
+      },
+      "collectMethod": "get",
+      "pagination": {
+        "type": "none"
+      },
+      "authentication": "login",
+      "timeout": 0,
+      "useRoundRobinDns": false,
+      "disableTimeFilter": false,
+      "rejectUnauthorized": false,
+      "safeHeaders": [],
+      "retryRules": {
+        "type": "backoff",
+        "interval": 1000,
+        "limit": 5,
+        "multiplier": 2,
+        "codes": [
+          429,
+          503
+        ],
+        "enableHeader": true
+      },
+      "loginUrl": "'https://your-salesforce-instance.com/services/oauth2/token'",
+      "loginBody": "`{ \"username\": \"${username}\", \"password\": \"${password}\" }`",
+      "tokenRespAttribute": "token",
+      "authHeaderKey": "Authorization",
+      "authHeaderExpr": "`Bearer ${token}`",
+      "clientSecretParamName": "client_secret",
+      "collectUrl": "'https://your-salesforce-instance.com/services/data/v58.0/query?q=SELECT,SourceIp,Status,UserId+FROM+LoginHistory'",
+      "collectRequestParams": [
+        {
+          "name": "et",
```

Comment:
Is there also a `lt` parameter? I'm not as familiar with the SFDC API...

Would be ideal to have a `lt` parameter so the latest time can also be populated should someone select that in the job config.